x-pack/platform/plugins/private/gen_ai_settings/server/plugin.ts

```diff
@@ -67,6 +73,34 @@ export class GenAiSettingsPlugin
       isDev: false,
     });
 
+    core.uiSettings.register({
+      /**
+       * TODO:
+       * Once assistants changes have been made, change setting to the following:
```

Comment:
Nit: Can we link the Kibana issue for the changes that have to be made in for the assistants here and the setting below? For future reference